l10n/po/ja_JP/_guides/kogito-drl.adoc.po

```diff
@@ -208,6 +215,13 @@ msgid ""
 "       this.deposit = deposit;\n"
 "   }\n"
 msgstr ""
+"   public LoanApplication(String id, Applicant applicant,\n"
+"                      \tint amount, int deposit) {\n"
+"       this.id = id;\n"
+"       this.applicant = applicant;\n"
+"       this.amount = amount;\n"
+"       this.deposit = deposit;\n"
+"   }\n"
 
```

Comment:
If you fill the translated text with empty string, original text is used as translated text.
So, if the source text and translated text is same, no need to copy & paste.